### PR TITLE
Dabbill canon 80D

### PR DESCRIPTION
**Pull request by dabbill**
Added basic Canon 80D support. Still need to fix pink hue and black area. Fixed tab spacing.

### CODE REVIEW

**Comment by dabbill** [comment]
Added crop for Canon 80D

**Comment by dabbill** [comment]
Added the black area for Canon 80D, seems to have fixed the pink hue

**Comment by dabbill** [comment]
Should be ready to merge with main branch. From what i can tell the black area and crop are now correct.

File: src/external/rawspeed/data/cameras.xml [hunk 1]
```diff
@@ -352,7 +352,29 @@
...
+    		<Crop x="264" y="34" width="0" height="0"/>
+    		<Sensor black="512" white="11892" iso_list="100 125"/>
+		<Sensor black="512" white="10749" iso_list="160"/>
+		<Sensor black="511" white="14338" iso_list="200 250"/>
+		<Sensor black="2048" white="10749" iso_list="320 640 1250 2500 5000"/>
+		<Sensor black="2048" white="14338" iso_list="400 500 800 1000 1600 2000 3200 4000 6400 16000"/>
+		<Sensor black="2046" white="14338" iso_list="8000 25600"/>
+		<Sensor black="2045" white="14338" iso_list="10000"/>
+		<Sensor black="2049" white="14338" iso_list="12800"/>
+		<BlackAreas>
```
Comment (LebedevRI) [inline src/external/rawspeed/data/cameras.xml]: Some of this lines are still not tab-indented only.
On first position is a space.

**Comment by dabbill** [comment]
I deleted all spaces in front of each line, and only used tab to get the space correct. Hope it sticks this time. :)

**Comment by dabbill** [comment]
This should be ready to be merged. mRAW is the only thing that is not working, but I am not sure if that is something fixed in DT or another library.

**Pull request merged**